src/geometry/mckinney_gammie_ryan.hpp

```diff
@@ -191,6 +202,9 @@ class McKinneyGammieRyan {
   Real x0_ = 0; // start point of smooth region
   Real smooth_ = 0.5;
   Real norm_;
+  Real hexp_br_ = 1000.;
+  Real hexp_nsq_ = 1.;
```

Comment:
As I discovered when I tried to run things with this value, nsq=1 is dangerous (it is the default in HARMPI and what was used in Tchekhovskoy '11 afaict though).
The first derivative of the function r(X1) is discontinuous for this value, and it exacerbates the problems with passing material over the break radius.  The effect may be much less if you use numerical first derivatives, as HARMPI does.

Reply:
Ah ok, I just pulled out the default values from kharma (as I interpreted them). Could you remind me what your preferred values are and I can update these defaults?

Reply:
:grin:

Reply:
Thanks -- updated to `hexp_nsq = 4`